packages/web-components/fast-foundation/docs/integrations/rollup.md

```diff
@@ -0,0 +1,260 @@
+keywords:
+  - rollup
+  - typescript
+---
+
+FAST works great with Rollup and TypeScript, using a standard setup. Let's take a look at how you can set up a FAST+Rollup+TypeScript project, starting from scratch.
+
+## Setting up the package
+
+First, let's make a directory for our new project. From the terminal:
+
+```shell
+mkdir fast-rollup
+```
+
+Next, let's move into that directory, where we'll set up our project:
+
+```shell
+cd fast-rollup
+```
+
+From here, we'll initialize npm:
+
+```shell
+npm init
+```
+
+Follow the prompts from npm, answering each question in turn. You can always accept the defaults at first and then make changes later in the `package.json` file.
+
+Next, we'll install the FAST packages, along with supporting libraries. To do that, run this command:
+
+```shell
+npm install @microsoft/fast-components @microsoft/fast-element tslib --save
+```
+
+We also need to install the Rollup build tooling:
+
+```shell
+npm install rollup typescript @rollup/plugin-typescript @rollup/plugin-node-resolve rollup-plugin-cleaner rollup-plugin-copy rollup-plugin-serve rollup-plugin-terser --save-dev
+```
+
+## Adding configuration and source
+
+Now that we've got our basic package and dependencies set up, let's create some source files and get things configured. Since we're going to be writing a bit of code, now is a great time to involve a code editor in the process. If you're looking for a great editor for TypeScript and front-end in general, we highly recommend [VS Code](https://code.visualstudio.com/).
+
+Open the `fast-rollup` folder in your favorite editor. You should see your `package.json` along with a `package-lock.json` and a `node_modules` folder.
+
+First, let's create a `src` folder where we'll put all our TypeScript code. In the `src` folder, add a `main.ts` file. You can leave the file empty for now. We'll come back to it in a bit.
+
+Then, in the root of your project folder, add a `tsconfig.json` file to configure the TypeScript compiler. Here's an example starter config that you can put into that file:
+
+```json
+{
+  "compilerOptions": {
+    "pretty": true,
+    "target": "ES2015",
+    "module": "ES2015",
+    "moduleResolution": "node",
+    "importHelpers": true,
+    "experimentalDecorators": true,
+    "declaration": true,
+    "declarationMap": true,
+    "sourceMap": true,
+    "noEmitOnError": true,
+    "strict": true,
+    "outDir": "dist/build",
+    "rootDir": "src",
+    "lib": [
+      "dom",
+      "esnext"
+    ]
+  },
+  "include": [
+    "src"
+  ]
+}
+```
+
+You can learn more about `tsconfig.json` options in [the official TypeScript documentation](https://www.typescriptlang.org/docs/handbook/tsconfig-json.html).
+
+:::note
+Do not set `useDefineForClassFields` to `true` in your `tsconfig.json` if you are using decorators. These two features conflict at present. This will be resolved in future versions of TypeScript and FAST.
```

Comment:
This note initially read a little oddly to me because you only mention *one* feature/config field in the sentence, implying the other. Maybe change this mention the `"ExperimentalDecorators"` field directly so the reader knows which *specific* fields conflict?

Reply:
Thank you, @nicholasrice - I will make the update!